forecast-inference/forecast_inference/app.py

```diff
@@ -251,12 +258,16 @@ def main(
     num_errors = len(pv_ids) - num_successes
 
     _log.info(
-        f"Ran successfully on {num_successes} PV sites ({num_successes / len(pv_ids) * 100:.1f} %) "
+        f"Ran successfully on {num_successes} PV sites ({num_successes / len(pv_ids) * 100:.1f}%)"
     )
-    _log.info(f"Errored on {num_errors} PV sites ({num_errors / len(pv_ids) * 100:.1f} %)")
+    _log.info(f"Errored on {num_errors} PV sites ({num_errors / len(pv_ids) * 100:.1f}%)")
+
+    # If requested, raise if any site failed
+    if raise_on_failure and num_errors > 0:
+        raise RuntimeError(f"{num_errors} PV site(s) failed out of {len(pv_ids)}")
 
-    # Raise an error if all forecasts fail
-    if num_successes == 0:
+    # Old default behavior
+    if not raise_on_failure and num_successes == 0:
```

Comment:
please retrun this to `if num_successes == 0`, as we always want to raise on error if none work